packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -164,6 +164,36 @@ export class Blockchain {
     return await this.client.call('getrawmempool', [verbose], 'bignumber')
   }
 
+  /**
+   * Get all in-mempool ancestors for a given transaction as string[]
+   *
+   * @param {string} txId the transaction id
+   * @param {boolean} verbose false
+   * @return {Promise<string[]>}
+   */
+  getMempoolAncestors (txId: string, verbose?: false): Promise<string[]>
+
+  /**
+   * Get all in-mempool ancestors for a given transaction as json object
+   *
+   * @param {string} txId the transaction id
+   * @param {boolean} verbose true
+   * @return {Promise<MempoolTx>}
+   */
+  getMempoolAncestors (txId: string, verbose?: true): Promise<MempoolTx>
+
+  /**
+   * Get all in-mempool ancestors if txId is in mempool as string[] if verbose is false
+   * else as json object
+   *
+   * @param {string} txId the transaction id
+   * @param {boolean} verbose default = false, true for json object, false for array of transaction ids
+   * @return {Promise<string[] | MempoolTx>}
+   */
+  async getMempoolAncestors (txId: string, verbose?: boolean): Promise<string[] | MempoolTx> {
```

Comment:
Let's use the default value `false` instead of sending `undefined`

```suggestion
  async getMempoolAncestors (txId: string, verbose: boolean = false): Promise<string[] | MempoolTx> {
```